### ISSUE TITLE
A question about label and output

### ISSUE THREAD

**Issue by pb585** [JJGO/UniverSeg#11]
Thanks for sharing, but I have two questions. The first is whether the foreground pixel value of the label is 1 or 0. The second is how to map the output from the script you provide to the mask, using normalization? Or softmax?

**Comment by JJGO** [comment]
- Foreground pixels should be 1 and background 0
- The model predicts logits, and output is binary. To get a continuous prediction you can use `torch.sigmoid` and to get a discrete prediction you can do `torch.sigmoid(yh) > 0.5` ( or any other threshold of your choosing). You can find an example in the [colab](https://colab.research.google.com/drive/1TiNAgCehFdyHMJsS90V9ygUw0rLXdW0r?usp=sharing) (search for sigmoid)
